client/containers/clusters/index.js

```diff
@@ -172,6 +183,26 @@ export default class ClustersContainer extends Component {
       this.setState({
         services: flatten(services)
       });
+      res.body.forEach(service => {
+        this.fetchTasks(cluster.clusterName, service.serviceName)
+      })
     }.bind(this));
   }
+
+  /**
+   * Fetch running tasks for the given `cluster` and `service`.
+   */
+
+  fetchTasks(cluster, service) {
+    request
+    .get(`/api/clusters/${cluster}/tasks/${service}`)
+    .end((err, { body }) =>{
+      if (err) {
+        const message = err.message || `unable to fetch tasks in cluster ${cluster.clusterName} family ${service} (${res.status})`;
```

Comment:
Doesnt this make more sense than line https://github.com/larseen/specs/blob/1f9bd9f953df068218f7b158c82abf7f1746fe4a/client/containers/clusters/index.js#L177
If you agree, I could add a PR to change it as well.

Reply:
Yes, this makes much more sense.